src/components/forms/inputs/TextInput.jsx

```diff
@@ -5,16 +5,23 @@ import InputBase from './InputBase';
 
 @injectIntl
 export default class TextInput extends InputBase {
+
+    constructor(props) {
+    super(props);
+    }
+
     renderInput() {
-        let placeholder;
+        let placeholder; 
 
         if(this.props.placeholder) {
             placeholder = this.props.intl.formatMessage(
                 { id: this.props.placeholder });
         }
 
+        const maxLength = this.props.maxLength ? this.props.maxLength : 0
```

Comment:
maxLength 0 means no characters allowed at all. Setting maxLength to null or undefined should do the trick.

Reply:
Have you actually tried this @niklasva82? It seemed to work for me.